app/jobs/visibility_copy_worker.rb

```diff
@@ -1,16 +1,9 @@
-class VisibilityCopyWorker
-  def queue_name
-    :permissions
-  end
-
-  attr_accessor :pid
-
-  def initialize(pid)
-    self.pid = pid
-  end
+class VisibilityCopyWorker < ActiveFedoraIdBasedJob
```

Comment:
With this class moving into `app/jobs/` should it be renamed from a worker to a job for consistency?